app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class RegistrationsController < Devise::RegistrationsController
+  layout 'internal'
+
+  protected
+
+  def after_update_path_for(_resource)
+    case resource
+    when Admin
+      admin_dashboard_path
+    when Student
+      student_dashboard_path
+    end
+  end
```

Comment:
@davithss you can improve your code

```ruby
def after_update_path_for
   return admin_dashboard_path if resource.is_a?(Admin)
   student_dashboard_path
end
```

Reply:
@basmoura Thank you for this suggestion but I think for this situation the case statement was easier to read and clear as well.